--- a/apps/crm/frontend/src/components/Apps.vue	2026-07-06 21:26:37.047832918 +0000
+++ b/apps/crm/frontend/src/components/Apps.vue	2026-07-06 21:26:37.062423221 +0000
@@ -3,7 +3,7 @@
     placement="right-start"
     trigger="hover"
     :hoverDelay="0.1"
-    :leaveDelay="0.1"
+    :leaveDelay="0.35"
   >
     <template #target="{ togglePopover }">
       <button
@@ -27,7 +27,7 @@
     </template>
     <template #body>
       <div
-        class="flex w-fit mx-2 min-w-32 max-w-48 flex-col rounded-lg border border-outline-gray-2 bg-surface-base p-1.5 text-sm text-ink-gray-8 shadow-xl auto-fill-[100px]"
+        class="flex w-fit min-w-32 max-w-48 flex-col rounded-lg border border-outline-gray-2 bg-surface-base p-1.5 text-sm text-ink-gray-8 shadow-xl auto-fill-[100px]"
       >
         <a
           v-for="app in apps.data"
