diff --git a/apps/crm/frontend/src/components/Apps.vue b/apps/crm/frontend/src/components/Apps.vue
index 6959a75..682e747 100644
--- a/apps/crm/frontend/src/components/Apps.vue
+++ b/apps/crm/frontend/src/components/Apps.vue
@@ -1,17 +1,11 @@
 <template>
-  <Popover
-    placement="right-start"
-    trigger="hover"
-    :hoverDelay="0.1"
-    :leaveDelay="0.1"
-  >
-    <template #target="{ togglePopover }">
+  <Popover bare side="right" align="start">
+    <template #trigger>
       <button
         :class="[
           active ? 'bg-surface-gray-3' : 'text-ink-gray-6',
           'group w-full flex h-7 items-center justify-between rounded px-2 text-base hover:bg-surface-gray-2',
         ]"
-        @click.prevent="togglePopover()"
       >
         <div class="flex gap-2">
           <AppsIcon class="size-4" />
@@ -25,9 +19,9 @@
         />
       </button>
     </template>
-    <template #body>
+    <template #default>
       <div
-        class="flex w-fit mx-2 min-w-32 max-w-48 flex-col rounded-lg border border-outline-gray-2 bg-surface-base p-1.5 text-sm text-ink-gray-8 shadow-xl auto-fill-[100px]"
+        class="flex w-fit min-w-32 max-w-48 flex-col rounded-lg border border-outline-gray-2 bg-surface-base p-1.5 text-sm text-ink-gray-8 shadow-xl auto-fill-[100px]"
       >
         <a
           v-for="app in apps.data"
